client/src/components/ProTx/components/dashboard/DashboardDisplay.js

```diff
@@ -49,167 +47,90 @@ function DashboardDisplay() {
     );
   }
 
+  const displayLayout = chartType => {
+    return (
+      <>
+        <DisplaySelectors
```

Comment:
The refactor here to displayLayout is problematic. It introduces three regressions that I can see where changing year is allowed in two views where they didn't want that and changing the rate/count in analytics.

I recognize that DisplaySelectors and approach here in DashboardDisplay should and can be refactored (lots of technical debt has been accrued here). but hard to review in this PR and easy to miss the regression.